javaagent-exporters/javaagent-exporters.gradle

```
@@ -2,6 +2,10 @@ plugins {
   id "com.github.johnrengelman.shadow"
 }
 
+ext {
+  mavenGroupId = 'io.opentelemetry.javaagent.exporters'
```

I think ok to put directly into `javaagent` groupId, there's not too many things there
```suggestion
  mavenGroupId = 'io.opentelemetry.javaagent'
```